core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -274,20 +272,4 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
       updateCache(resource, acls)
     }
   }
-
-  object ZkStateChangeListener extends IZkStateListener {
-
-    override def handleNewSession() {
-      aclChangeListener.processAllNotifications
-    }
-
-    override def handleSessionEstablishmentError(error: Throwable) {
-      fatal("Could not establish session with zookeeper", error)
-    }
-
-    override def handleStateChanged(state: KeeperState) {
-      //no op
```

Comment:
Is it better to log state changes as INFO or even DEBUG so that we know what's going with the session and connection?